portfolio/src/main/webapp/index.html

```diff
@@ -9,6 +9,9 @@
   <body onload="getjson()">
     <div id="content">
       <h1>Welcome to Alec's portfolio!</h1>
+      <p>Click here to login:</p>
+      <button onclick="getLoginStatus()">Login</button>
+      <div id="greeting-container"></div>
```

Comment:
You used `loginstatus-container` as ID in your JS code, but you are using a different ID string here, which will probably cause some issues.